src/tests/unit/tests/injected/visualization/svg-drawer.test.ts

```diff
@@ -701,14 +767,16 @@ describe('SVGDrawer', () => {
         drawerUtilsMock.verifyAll();
 
         expect(circles.length).toBe(2);
-        expect(lines.length).toBe(1);
+        expect(lines.length).toBe(0);
```

Comment:
Confused why we expect 0 lines when the test says "with solid line"?

Reply:
Is this related to the undesired behavior mentioned in the description?

Reply:
I need to check on this again but this seems to be a part of the weird diff behavior: I didn't modify this test case but it makes it seem like I did because of the test deletion.